src/nrtk_explorer/app/transforms.py

```diff
@@ -118,21 +124,89 @@ def delete_meta_state(old_ids, new_ids):
         self.state.transforms = [k for k in self._transforms.keys()]
         self.state.current_transform = self.state.transforms[0]
 
-        # Transform enabled control ##
-        self.state.transform_enabled = True
+        # Annotations enabled control via predictions_original_images_enabled ###
```

Comment:
I guess I don't like the original code. But you are keeping the same pattern as before. It could be ok for now... But I will definitely change that down the road.